model/src/lib.rs

```diff
@@ -100,6 +100,12 @@ impl TokenPair {
     }
 }
 
+impl Default for TokenPair {
```

Comment:
ubernit: Conceptually, it doesn't really make sense to me for `TokenPair` to implement `Default` since this default isn't really valid and is really only useful for tests (maybe the implementation should be `#[cfg(test)]`?). However, this is very nit-picky of me so feel free to leave as is.